### DISCUSSION TITLE
Grafana 8.0 has broken a few things

### DISCUSSION THREAD

**Discussion by 3rdStng** [teslamate-org/teslamate#1703] [General]
EDIT: I am 100% sure this is all related to the update. I forced my Grafana docker back to 7.5.7 and everything is working.

Grafana 8.0 was released and it breaks a few things within TM. My docker upgraded to it, without me knowing, and now I have a few dashboards that give errors. 
There were about 4 plugins I needed to reinstall. All but one (grafana-map-panel) installed. This one doesn't appear in the repository.
Dashboard errors/issues:
* Charge Level - error of "convert frame from rows error: sql: Scan error on column index 2, name "Usable Battery Level [%]": converting NULL to int16 is unsupported" 
* Charging Stats - grafana-map-panel plugin is missing, so the maps and pie charts don't appear
* Timeline - error of "convert frame from rows error: sql: Scan error on column index 14, name "SoC": converting NULL to int16 is unsupported" 
* Trip - The map doesn't update. 
* Trip - Error in the Drives section of "convert frame from rows error: sql: Scan error on column index 10, name "duration_min": converting NULL to int16 is unsupported"
* Trip - Error in the Charges section of "convert frame from rows error: sql: Scan error on column index 8, name "duration_min": converting NULL to int16 is unsupported"
* Visited - Map doesn't populate.

**Comment by bpankz** [comment]
panodata-map-panel-0.16.0 released yesterday, still not showing correct maps with grafana 8 (maybe because it's unsigned?). downgrading to grafana 7.5.10 works as intended. plugin name has changed https://github.com/panodata/panodata-map-panel/releases/tag/0.16.0

**Reply by miguelangelnieto** [reply]
Thanks. I didn't know why my maps were showing no data. Downgrading worked!
